python/ray/train/lightning/lightning_predictor.py

```diff
@@ -0,0 +1,64 @@
+import logging
+from typing import Optional, Type
+
+from ray.air.checkpoint import Checkpoint
+from ray.data.preprocessor import Preprocessor
+from ray.train.lightning.lightning_checkpoint import LightningCheckpoint
+from ray.train.torch.torch_predictor import TorchPredictor
+from ray.util.annotations import PublicAPI
+import pytorch_lightning as pl
+
+logger = logging.getLogger(__name__)
+
+
+@PublicAPI(stability="alpha")
+class LightningPredictor(TorchPredictor):
+    """A predictor for PyTorch Lightning modules.
+
+    Args:
+        model: The PyTorch Lightning module to use for predictions.
+        preprocessor: A preprocessor used to transform data batches prior
+            to prediction.
+        use_gpu: If set, the model will be moved to GPU on instantiation and
+            prediction happens on GPU.
```

Comment:
Can we add a tested code example in this docstring?

Reply:
Added!